src/storage/storage.cc

```diff
@@ -769,6 +770,98 @@ rocksdb::Status Storage::FlushScripts(engine::Context &ctx, const rocksdb::Write
   return Write(ctx, options, batch->GetWriteBatch());
 }
 
+StatusOr<int> Storage::IngestSST(const std::string &sst_dir, const rocksdb::IngestExternalFileOptions &ingest_options) {
+  std::vector<std::string> sst_files;
+  DIR *dir = opendir(sst_dir.c_str());
+  if (!dir) {
+    return {Status::NotOK, "Failed to open directory " + sst_dir};
+  }
+
+  struct dirent *entry = nullptr;
+  while ((entry = readdir(dir)) != nullptr) {
+    std::string filename = entry->d_name;
+    if (filename.length() >= 4 && filename.substr(filename.length() - 4) == ".sst") {
+      sst_files.push_back(sst_dir + "/" + filename);
+    }
+  }
+  closedir(dir);
+
+  if (sst_files.empty()) {
+    LOG(WARNING) << "No SST files found in " << sst_dir;
+    return 0;
+  }
+
+  // Create a map to store SST files for each column family
```

Comment:
I think this comment and some of the comments below are redundant. 

**Good code is comments.**

Reply:
addressed via 4ac013680a46ec33cdddeb9705b4ab55fb75dde2